packages/labs/ssr/src/test/integration/tests/basic.ts

```diff
@@ -4825,6 +4825,88 @@ export const tests: {[name: string]: SSRTest} = {
     };
   },
 
+  'LitElement: Attribute binding (mixed case)': () => {
```

Comment:
Is it possible to add a test case where attribute casing is preserved? E.g. svg element in a render template?

Reply:
I added it separately as a unit test that makes sure the rendered markup has attribute casing preserved. How the browser handles that is up to the browser.

Reply:
What is making you suspect something?

Reply:
I tried some combinations in the playground and found inconsistent results. I'll investigate and document in a separate issue. :)